### DISCUSSION TITLE
<créate a forum>

### DISCUSSION THREAD

**Discussion by jondalar24** [community/community#129734] [Programming Help]
### Select Topic Area

General

### Body

Hi everyone,

I'm working on a forum application for my project and would appreciate any advice or insights. Specifically, I want to create a dynamic forum that can be extended for other features later.

1. What technologies or frameworks would you recommend for building a forum?
2. Are there any best practices for handling continuous monitoring of forum posts?
3. Any suggestions for finding a free repository to use as a base for the whole project?

Looking forward to your suggestions! 🙌

**Comment by ebndev** [comment]
Hey @jondalar24, thanks for being a part of the GitHub Community!

How scalable do you want this to be? You could definitely build something with CRUD functionality using the [MERN stack](https://github.com/davidsaulrodriguez/mern-stack-template) and using npm packages like [Passport](https://www.youtube.com/watch?v=_lZUq39FGv0) to handle user authentication. 

Beyond CRUD functionality, you'd likely have to integrate third party apps or APIs if you wanted to automate monitoring forum posts or detecting when to delete / block users.

**Reply by jondalar24** [reply]
Hello, thank you for the welcome. I am working on my Final Degree Project and what I am studying is to create an automatic manager that can read entries from both users and the master’s degree. That’s why I need, as a laboratory, a forum code where I can implement this automatic user, that’s why I was asking for help to get started. I’m a bit lost since in the degree we study a lot of syntax, but this is my first comprehensive project.
